proto/storage/product_usage.proto

```diff
@@ -0,0 +1,26 @@
+syntax = "proto3";
+
+package storage;
+
+import "gogoproto/gogo.proto";
+import "google/protobuf/timestamp.proto";
+
+option go_package = "storage";
+
+// SecuredUnits represents a record of an aggregated secured clusters usage
+// metrics. The metrics are aggregated periodically, and put into the database.
+message SecuredUnits {
+    // id is not used to retrieve data, but serves mostly for compatibility with
+    // the current implementation of the query generator.
+    string id                           = 1 [(gogoproto.moretags) = 'sql:"pk,type(uuid)"'];
+    // timestamp stores the moment at which the values of the metrics below are
+    // aggregated.
+    google.protobuf.Timestamp timestamp = 2;
```

Comment:
What is the rationale behind not making this a column such that we could use queries instead of walking the data?